src/Directives/DefaultDirective.php

```diff
@@ -6,7 +6,7 @@
 
 use Mazedlx\FeaturePolicy\Directive;
 use Mazedlx\FeaturePolicy\DirectiveContract;
-use Mazedlx\FeaturePolicy\Exceptions\InvalidDirective;
+use Mazedlx\FeaturePolicy\Exceptions\UnsupportedPermissionException;
```

Comment:
this relates to an permission that either does not exist, or exists in another group